### PR TITLE
Allowing to skip schema load

### PR DESCRIPTION
**Pull request by paganotoni**
In some situations, the database user configured in the database connection is not the owner of the plpgsql extension, and Postgres ends up saying when we try to load the schema file.

```
Error: unable to load schema for application_test: pq: must be owner of extension plpgsql
```

This PR allows to skip schema load and go straight to migrations.

### CODE REVIEW

File: buffalo/cmd/test.go [hunk 1]
```diff
@@ -22,9 +22,12 @@ import (
 const vendorPattern = "/vendor/"
 
 var vendorRegex = regexp.MustCompile(vendorPattern)
+var skipSchemaLoad = false
 
 func init() {
 	decorate("test", testCmd)
+	testCmd.Flags().BoolVarP(&buildOptions.ExtractAssets, "skip-schema-load", "s", false, "skips loading the schema and instead runs migrations before running tests")
```
Comment (markbates) [inline buffalo/cmd/test.go]: I'm not a fan of the name of this flag. Any other options?

Comment (paganotoni) [reply buffalo/cmd/test.go]: Oh boy! ouch! going for it.

Comment (paganotoni) [reply buffalo/cmd/test.go]: @markbates  what about "forceMigrations" and -m --force-migrations ?

**Comment by paganotoni** [comment]
@markbates moved the variable to be `forceMigrations` (-m and --force-migrations in the cli).

**Pull request merged**